package.json

```diff
@@ -3,9 +3,7 @@
   "description": "Use SCORM standard package with Tailor",
   "author": "ExtensionEngine <info@extensionengine.com>",
   "license": "MIT",
-  "version": "0.0.1",
-  "main": "dist/tce-scorm.js",
-  "module": "dist/tce-scorm.esm.js",
+  "version": "0.0.2",
```

Comment:
Revert this change, version will be set after PR is merged, to be inline with other packages.

Reply:
Reverted.